tests/integration/operator/operator_test.go

```diff
@@ -244,6 +246,20 @@ spec:
 	})
 })
 
+func ensureIstioCRsDeleted() error {
```

Comment:
why "ensure"? This deletes the CRs, so I'd name it "deleteIstioCRs".

Also, "CRs" can easily be misread as "CRDs", so it will probably cause people to think this is deleting the CRDs. This just happened to me right now. So, I propose using the name "resources" instead of "CRs" everywhere (function name, variables, etc.).